### PR TITLE
[luci/test] Introduce evalverifytol script

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce evalverifytol.sh script to support tolerance with value test.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #8203 , from draft #8279
To solve #8340

File: compiler/luci-value-test/evalverifytol.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# bin_dir  : build directory of luci-value-test (ex: build/compiler/luci-value-test)
+# work_dir : artifacts directoy where test materials exist
+# venv_dir : python virtual environment home directory
+
+VERIFY_SOURCE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+VERIFY_SCRIPT_PATH="${VERIFY_SOURCE_PATH}/luci_eval_verifier.py"
+BINDIR="$1"; shift
+WORKDIR="$1"; shift
+VIRTUALENV="$1"; shift
+INTERPRETER_DRIVER_PATH="$1"; shift
```
Comment (mhs4670go) [inline compiler/luci-value-test/evalverifytol.sh:18]: Does this variable exist in an interface?
```bash
./evalverifytol.sh <path/to/bin_dir> <path/to/work_dir> <path/to/venv_dir> \
#                    <TEST 1> <RTOL 1> <ATOL 1> <TEST 2> <RTOL 2> <ATOL 2> ...
```

Comment (seanshpark) [reply compiler/luci-value-test/evalverifytol.sh:18]: Good catch :) Need to revise `evalverify.sh` too.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by YongseopKim — APPROVED** [review]
LGTM

**Review by BalyshevArtem — APPROVED** [review]
LGTM, thank you

**Pull request merged**